src/arbeidssøkerskjema/utils/innsending.ts

```diff
@@ -0,0 +1,10 @@
+import { IArbeidssøker } from '../../models/steg/aktivitet/arbeidssøker';
+
+export const mapDataTilLabelOgVerdiTyper = (skjema: IArbeidssøker) => {
+  const nyttSkjemaObjekt: any = {};
+
+  for (const [key, value] of Object.entries(skjema)) {
```

Comment:
Har vi polyfill på Object.entries for IE11 ?

Reply:
Yup, har en` import babel-polyfill` i starten av index.tsx :)